src/rp2040/hardware_regs/include/hardware/platform_defs.h

```diff
@@ -37,7 +37,15 @@
 
 #define PIO_INSTRUCTION_COUNT _u(32)
 
-// PICO_CONFIG: XOSC_MHZ, The crystal oscillator frequency in Mhz, type=int, default=12, advanced=true, group=hardware_base
+// For USB operation this *has* to be 48 MHz
+#define USB_CLK_MHZ _u(48)
```

Comment:
is this 100% true; i mean could you set it a little off

Also i wonder if we should specify this and SYS_CLK in khz

Reply:
> Also i wonder if we should specify this and SYS_CLK in khz

I guess i meant, "let's specify these in khz" since i don't think we want to force people to use multiples of Mhz

Reply:
Also, given that these are ultimately derived from XOSC_MHZ, perhaps it makes sense to list them in this file _atfer_ `XOSC_MHZ` rather than before?

Reply:
i was referring to USB_CLK_MHZ and SYS_CLK_MHZ (github may have been confusing here as i clicked on USB_CLK_MHZ but it shouws a comment about XOSC_MHZ)

Can XOSC actually be specified in non MHZ increments? (I had assumed not)